test/test_torch.py

```diff
@@ -14662,6 +14662,180 @@ def test_ones_like_multiple_device(self, devices):
         self.assertEqual(output, expected)
 
 
+# Tests ops and indexing to ensure they return views (and new tensors) as
+# appropriate.
+# Note: these tests do not verify tensor storage is the same or different
```

Comment:
Sounds good.
They should still populate the `._is_view()` and `._base` no?

Reply:
Aha! I didn't even realize we had ._is_view(), and it turns out XLA tensors do implement it! That's much better.

Reply:
But I would argue you want both the `._is_view()` AND the check that it changes the value inplace. We have history of doing one but not the other and it is **really** hard to fix afterwards.

Reply:
Interesting. So you think we extend each with an ._is_view() check? Sounds good.